Kimjin/3주차 과제/과제1/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Kim Jin</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="box1">
+      <h1>눌러봐요</h1>
+    </div>
+    <div class="box2">
+      <div class="one1"></div>
+      <div class="one2"></div>
+      <div class="one3"></div>
+      <div class="one4"></div>
+      <div class="one5"></div>
```

Comment:
여기 클래스명들도 조금 더 의미 있게 해주면 좋을 것 같아요.
`one1`, `one2`, ...보다는 `circle` 과 같은 이름이 좋을 것 같고, css를 보면 `one4`에만 효과가 들어가서 스타일이 다른데 그럼 다른 것들과 클래스명을 구분하면 좋을 것 같아요.
예를 들어 효과가 없는 것들은 `circle`, 효과가 있는 것은 `point-circle` 과 같이요!
`point-circle`은 기본 `circle`과 모양은 같고, 눌렀을 때 효과만 다른 거니까 `circle` 클래스도 있어야겠네요!
```suggestion
      <div class="circle"></div>
      <div class="circle"></div>
      <div class="circle"></div>
      <div class="circle point-circle"></div>
      <div class="circle"></div>
```